features/readme.md

```diff
@@ -1,5 +1,4 @@
 [![Travis status](https://secure.travis-ci.org/zipmark/rspec_api_documentation.png)](https://secure.travis-ci.org/zipmark/rspec_api_documentation)
-[![Gemnasium status](https://gemnasium.com/zipmark/rspec_api_documentation.png)](https://gemnasium.com/zipmark/rspec_api_documentation)
```

Comment:
Don't know what this is doing here, to be honest. This looks like an old version of README.md that was copied in this folder? Maybe it had a purpose? 🤔

Reply:
Yeah let's remove this file entirely.

Reply:
Oh, there it is!
https://github.com/zipmark/rspec_api_documentation/commit/a939f26c64239fabcc974e2ed17dba84247628ef

> Add readme for relishapp

This README file ends up here: https://relishapp.com/zipmark/rspec-api-documentation/docs

Wonder if the README file in this folder should be simply synced with the repo's main README file? Maybe using a symlink?

Reply:
👍 ok then. we keep it for now, will poke around to relish app also as I didn't set it up.